macros/invoke_core_deployer.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -o nounset
```

Comment:
rm

macros are often sourced and should not modify the environment of the parent script